walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -79,15 +94,15 @@
    */
   public Collection<TimeRange> queryHelper(Collection<Event> separateEvents, MeetingRequest request) {
     // Map each {@code Event} to it's {@code TimeRange}.
-    List<TimeRange> TimeRangeList = separateEvents.stream()
-                     .map(a -> a.getWhen())
+    List<TimeRange> timeRangeList = separateEvents.stream()
+                     .map(Event::getWhen)
                      .collect(Collectors.toList());
 
     // Resulting Collection
     Collection<TimeRange> openTimeRanges = new ArrayList();
 
     int currentMinTime = 0;
```

Comment:
Iiuc, this variable represents the earliest start time. How about name it something along those lines? E.g. earliestStartTime, earliestOpenStartTime, earliestViableStartTime?  I kinda like 'early; since it has temporal significance as opposed to minimum time which has more of a duration connotation when I read it. Lmk if I missed anything

Reply:
I understand you. I changed it.